nilearn/maskers/tests/test_masker_validation.py

```diff
@@ -88,7 +88,7 @@ def test_check_embedded_nifti_masker():
         masker = _check_embedded_nifti_masker(
             owner, multi_subject=multi_subject
         )
-        assert type(masker) == type(mask)
+        assert type(masker) == type(mask)  # noqa : E721
```

Comment:
am unsure how to best fix this one

Reply:
Isn't that equivalent to `assert isinstance(masker, type(mask))`?

Reply:
that incantation pacified the linting gods on my machine, let's see if this works on the remote gods of the CI too

Reply:
fixed
the gods are happy